src/solvers/nonlinear_solid_operators.hpp

```diff
@@ -74,6 +74,50 @@ class NonlinearSolidQuasiStaticOperator : public mfem::Operator {
  *  are given vectors, and dt is a scalar.S
  */
 class NonlinearSolidReducedSystemOperator : public mfem::Operator {
+public:
+  /**
+   * @brief Construct a new Nonlinear Solid Reduced System Operator object
+   *
+   * @param[in] H_form The nonlinear stiffness form
+   * @param[in] S_form The linear viscosity form
+   * @param[in] M_form The linear mass form
+   * @param[in] ess_bdr The essential boundary conditions
+   */
+  NonlinearSolidReducedSystemOperator(const mfem::ParNonlinearForm& H_form, const mfem::ParBilinearForm& S_form,
+                                      mfem::ParBilinearForm&                       M_form,
```

Comment:
const?

Reply:
Unfortunately we have to call `ParallelAssemble` on the mass matrix, so this object is actually not const. The other forms just need to be evaluated in a matrix-free fashion.